Translations/glossary.md

```diff
@@ -0,0 +1,66 @@
+# Translations glossary
+
+## Variables
+
+There are three types of variables in the text files: %s, %i, %d. All of these will add something within the single string, here are some examples. 
+
+`%s` will always be another word. Example: `%s of %s` refers to an created Item which includes affixes. `Tin Cap of the Fool` would be such an item.
+`%i` will always be a number. Example: Fire hit damage: %i-%i` will indicate how much fire damage a weapon will deal
+`%d` will always be a number. Example: `Player '%s' (level %d) just joined the game` - Player Arcas (level 15) just joined the game
+
+**IMPORTANT**: You have to keep those intact and in the same order. 
+
+Example: 
+
+Working:
+
+	`%s' (level %d) just joined the game`
+changed to
+	`%s' (level %d) is now with you.`
+
+Not working:
+
+	`%s' (level %d) just joined the game`
+changed to
+	`A level %d Character named % s' is now with you.`
+	
+## Names
+
+Please try to keep original names intact, however it's up to your judgment to change something that sounds completely wrong in the language you are translating to.
+
+## Coherence
+
+Make sure that you use the same terminology where it's applicable. 
+
+Example:
+
+Good:
+	`fully recover life`
+	`recover life`
+	`fully recover mana`
+	`recover mana`
+
+Bad:
+	`recover all life points`
+	`recover partial life`
+	`fully recover mana`
```

Comment:
Sorry but why is it different from line 40? Is it a copy+paste typo?

Reply:
Line 39 / 45 are inconsistent. Funnily enough this is an issue with vanilla. ;)